lcs/agents/acs2er/ACS2ER.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import random
+from lcs import Perception
+from lcs.agents.Agent import TrialMetrics
+from lcs.agents.acs2er.ReplayMemory import ReplayMemory
+from lcs.agents.acs2er.ReplayMemorySample import ReplayMemorySample
+from lcs.strategies.action_selection.BestAction import BestAction
+from lcs.agents.acs2 import ClassifiersList
+from lcs.agents.acs2 import Configuration
+from lcs.agents.Agent import Agent
+
+logger = logging.getLogger(__name__)
+
+
+class ACS2ER(Agent):
+
+    def __init__(self,
+                 cfg: Configuration,
+                 population: ClassifiersList = None,
+                 replay_memory: ReplayMemory = None) -> None:
```

Comment:
I'd rather prefer to keep agent constructors taking only the `cfg` and `population` parameters. The replay memory size can later be fetched from `self.cfg`

Reply:
done